.github/workflows/verify-api.yaml

```diff
@@ -24,13 +24,20 @@ jobs:
       - name: Run api in docker container
         run: |
           docker buildx build -t epp-biorxiv-xslt:api --target api .
-          docker run -d -p 3000:80 epp-biorxiv-xslt:api
-          sleep 5 # wait for the container to start up
+          docker run -d --name api-container -p 3000:80 epp-biorxiv-xslt:api
+
+      - name: Verify that api is healthy
+        run: |
+          sleep 60 # wait for the container to start up
```

Comment:
This starting of the container consistently takes > 36s on my machine, which is quite long. GHA can be very variable in performance, and 60s maybe too short (or indeed, could be too long).

docker-compose gives a --wait command, which might be worth considering:
```sh
> cat docker-compose.yaml 
services:
  api:
    image: epp-biorxiv-xslt:api

> docker buildx build -t epp-biorxiv-xslt:api --target api . --load
[+] Building 36.1s (27/27) FINISHED                                                             docker-container:musing_swartz
# <snip build output>


>> docker-compose up --wait
[+] Running 2/2
 ✔ Network enhanced-preprints-biorxiv-xslt_default  Created                                                               0.0s 
 ✔ Container enhanced-preprints-biorxiv-xslt-api-1  Healthy                                                              36.2s 

```

Reply:
Sounds like a great improvement. Will add it in.